docs/06-manual-builds.md

```diff
@@ -1,132 +1,121 @@
-
-## Manually building for iOS
-
-### Setup
-
+# Manually Running and Building the App
+If you want to run the app locally or create `.ipa` or `.apk` builds for testing purposes you can follow this guide.
+If you only want to quickly push an update to the stores then take a look at the [CI/CD documentation](08-cicd.md#triggering-a-build-in-ci).
+
+## Contents
+* iOS
+    * [Setup and prerequisites](#setup-on-ios)
+    * [Running the app](#running-the-app-on-ios)
+    * [Building the app with a distribution certificate](#building-the-app-with-a-distribution-certificate)
+* Android
+    * [Setup and prerequisites](#setup-on-android)
+    * [Running the app](#running-the-app-on-android)
+    * [Building the app using a test signing keystore](#building-the-app-using-a-test-signing-keystore-without-fastlane)
+    * [Building the app using for the Play Store](#building-the-app-for-the-play-store)
+
+## iOS
+### Setup on iOS
 #### Prerequisites
-* Install and setup XCode
-* [Install and setup Fastlane](08-cicd.md#setup-of-fastlane)  (Necessary for certificate management)
-* Read about [Apple Certificates](10-apple-certifcates.md)
-
-#### Setup CocoaPods
-
-CocoaPods is used for dependency management of the native libraries.
-First you have to make sure that Ruby is working correctly. We advise to use RVM as mentioned in the [setup of Fastlane](08-cicd.md#setup-of-fastlane). After running this setup you should also be able to do `bundle exec pod --version`.
-    
-*Note: `bundle exec pod install` uses the versions from the `Podfile.lock`. `bundle exec pod update` updates the `Podfile.lock`.*
-
-
-#### Setup Certificates
-
-First you have to prepare a [few environment variables](08-cicd.md#environment-variables-and-dependencies):
+* Install and setup XCode.
+* [Install and setup Fastlane](08-cicd.md#fastlane-setup).
+* Read about [Apple Certificates](10-apple-certifcates.md).
 
+#### Certificates Setup
+* Prepare the necessary environment variables:
 ```bash
 export FASTLANE_USER=<secret>
 export FASTLANE_PASSWORD=<secret>
 export MATCH_PASSWORD=<secret>
 ```
 
-Finally, install the certificates locally:
+More information about the necessary environment variables can be found [here](08-cicd.md#environment-variables-and-dependencies).
 
+* Install the certificates locally:
 ```bash
-cd ios/
-bundle exec fastlane certificates
+cd ios && bundle exec fastlane certificates
 ```
 
-*Note: After setting up the certificates you can also start to use XCode to build the app.*
+#### Dependency Management
+CocoaPods is used for dependency management of the native libraries.
+It should be usable after setting up Fastlane.
 
-### Run the app in a Simulator
+* Make sure CocoaPods is installed by running `bundle exec pod --version`.
```

Comment:
Please add a note about using RVM and Bundler for installing  CocoaPods. This can be done using `bundle install --path vendor/bundle`

Reply:
The link to fastlane (`[setup of Fastlane](08-cicd.md#setup-of-fastlane)`) as helpful as the setup of RVM is descriped there.

Reply:
Why? This is covered in the setup section above?

Reply:
Oops did not see it :heavy_check_mark: